metapool/sample_sheet.py

```diff
@@ -10,6 +10,13 @@
                                REVCOMP_SEQUENCERS)
 from metapool.plate import ErrorMessage, WarningMessage, PlateReplication
 
+BIOINFORMATICS_KEY = 'Bioinformatics'
+CONTACT_KEY = 'Contact'
+SAMPLE_CONTEXT_KEY = 'SampleContext'
+SAMPLE_NAME_KEY = "Sample Name"
```

Comment:
I am inclined to leave "SampleContext" as Pascal case because it is a section header in the sample sheet, and all the others begin with upper case letters (also, I think it would be disastrous to lcase those section headers since there are tons of existing past sample sheets out there that have the upper-case text written into them).  However, I will more than happily change "Sample Name" and the other headers *within* in the totally-new SampleContext section to be snake case ... I was just attempting to follow the apparent convention within in the other sections :)

Reply:
IMHO, I think keeping the convention in the other sections is the most important.

Reply:
@charles-cowart wait, so does that mean that you *don't* want me to change the case of the column names inside the new SampleContext section? I note that there isn't a single convention within the other sections ... e.g., the [Bioinformatics] section includes columns named `Sample_Project`, `QiitaID`, and `contains_replicates` :-O

Reply:
I see what you're saying. Specifically, the column names in the new section are:
"Sample Name"
"Sample Type"
"Primary Qiita Study"
"Secondary Qiita Studies"

I can accept it as-is. I have a personal preference for replacing the spaces with '_' but I think everyone in the lab is a little different, hence 'Sample_Project', 'QiitaID', and 'contains_replicates'. Yeah it's a mess. It would be nice to rename them all for consistency. Some are just legacy and what people are used to, while others is whoever requested or programmed the feature.

Perhaps what's needed down the road is a vote from the entire lab what case to make standard, if any. 😅

Reply:
Left "SampleContext" as is to match other section names, changed all internal headers in the new section to be snake case.